pkg/ui/workspaces/cluster-ui/src/activeExecutions/activeStatementUtils.ts

```diff
@@ -168,6 +168,14 @@ export function getActiveExecutionsFromSessions(
   };
 }
 
+export function getExecututionStatuses(): string[] {
+  const execStatuses: string[] = [];
+  for (const execStatus in ExecutionStatus) {
+    execStatuses.push(execStatus);
+  }
+  return execStatuses.sort();
+}
+
```

Comment:
nit: instead of a function just create the array directly. We can always spread it if we need a copy.
```
const execStatusList = Object.values(ExecutionStatus);
```